### DISCUSSION TITLE
erro ao usar IResponse em outra ou na mesma request

### DISCUSSION THREAD

**Discussion by eduardofr275** [viniciussanchez/RESTRequest4Delphi#154] [General]
quando eu chamo uma Request duas ou mais vezes na mesma procedure gera erro, por exemplo
LRes:=TRequest.New.BaseURL('http://*****.fcgi')
    .AddParam('session',sessao)
    .Accept('application/json')
    .AddBody(usu)
    .DataSetAdapter(FDMemTable2)
    .post;
se eu uso sem o IResponse funciona, acredito que fica algum objeto amarrado na memória
TRequest.New.BaseURL('http://*****.fcgi')
    .AddParam('session',sessao)
    .Accept('application/json')
    .AddBody(usu)
    .DataSetAdapter(FDMemTable2)
    .post;

**Comment by viniciussanchez** [comment]
Olhando o seu exemplo acima não consegui entender onde está o problema. 
Coloque um código aqui de exemplo que gere o erro por favor. E também um print do erro que está gerando para facilitar.
